docker/maintenance/SMW_conceptCache/SMW_conceptCache-1318.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: maintenance-media-wiki
+spec:
+  schedule: "0 10 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: logger
+            image: artifactory.wikia-inc.com/sus/mediawiki-logger:latest
+          - name: php
+            image: artifactory.wikia-inc.com/sus/mediawiki-sandbox-php:54f12b1.9306b50
```

Comment:
We need to keep this tag in sync with production deploy. Should we make this a template that Jenkinsfile will use and pass a correct tag?